variables.tf

```diff
@@ -3,6 +3,11 @@ variable "naming_rules" {
   type        = string
 }
 
+variable "dnsserver" {
+  description = "custom dns server" 

```

Comment:
Please set a more explicit description, e.g. "A list of custom DNS servers to use inside your virtual network".

Reply:
changed